package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metadaoproject/futarchy-sdk",
-  "version": "1.1.0-alpha.31",
+  "version": "1.1.0-alpha.32",
```

Comment:
Can you confirm for me that when you do a pnpm install and then carry this into the FE it's working as expected, we had an issue today where we had previously had `^2.1.0` which installed `2.2.0` and was breaking everything. So this is now constrained to the non-breaking version and I just want to make sure the SDK is functioning as expected.